docs/generated/oc_by_example_content.adoc

```diff
@@ -920,6 +933,32 @@ Create a service account with the specified name.
 ====
 
 
+== oc create user
+Create a user.
+
+====
+
+[options="nowrap"]
+----
+  # Create a user with the username "ajones" and the display name "Adam Jones"
+  $ oc create user ajones --full-name="Adam Jones"
+----
+====
+
+
+== oc create useridentitymapping
```

Comment:
I am not sure if this is the best way to expose this feature to users. Are we expecting users to create identities and not map them? When are users creating identities and not using identity providers? I'm not against having these commands in the CLI as I'm sure there are a lot of complex use cases that they enable, but I wonder if we need something different for Joe User.